test/test_v4_secscan.py

```diff
@@ -0,0 +1,131 @@
+import unittest
+import pytest
+
+from app import app, instance_keys, storage
+from initdb import setup_database_for_testing, finished_database_for_testing
+from util.secscan.v4.api import ClairSecurityScannerAPI, IncompatibleAPIResponse
+from util.secscan.v4.fake import fake_security_scanner
+from workers.securityworker.securityworker import SecurityWorker
+from data.registry_model import registry_model
+from data.secscan_model import secscan_model
+
+
+def manifest_for(namespace, repository, tagname):
+    repository_ref = registry_model.lookup_repository(namespace, repository)
+    tag = registry_model.get_repo_tag(repository_ref, tagname, include_legacy_image=True)
+
+    return registry_model.get_manifest_for_tag(tag, backfill_if_necessary=True)
+
+
+class TestV4SecurityScanner(unittest.TestCase):
```

Comment:
We should use the unit test TestCase anymore;  instead, we use pytest now

Reply:
Done